atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/DbAtlasDbFactory.java

```diff
@@ -24,10 +25,11 @@
 import com.palantir.timestamp.PersistentTimestampService;
 import com.palantir.timestamp.TimestampService;
 
+@AutoService(AtlasDbFactory.class)
 public class DbAtlasDbFactory implements AtlasDbFactory {
     @Override
     public String getType() {
-        return "db";
+        return "postgres";
```

Comment:
No, we can refactor this without change to the config layout to have it support oracle.